advanced_guides/binary.md

```diff
@@ -1,41 +1,48 @@
 # MeiliSearch Binary
 
+## Compiling from source
 
-## How to compile meiliSearch
+MeiliSearch is made in `Rust`. Therefore the Rust toolchain must [be installed](https://www.rust-lang.org/tools/install) to compile the project.
 
-### Dependencies
-
-MeiliSearch is made in `rust`. Because of that `rust` and its package manager `cargo` must [be installed](https://www.rust-lang.org/tools/install) to compile the project.
-
-### Compiling
-
-Now that the Rust toolchain is installed, we need to git clone and go to the cloned directory.
+If you have the Rust toolchain already installed, you need git clone and go to the cloned directory.
 
 ```bash
-git clone https://github.com/meilisearch/MeiliDB
-cd MeiliDB
+git clone https://github.com/meilisearch/MeiliSearch
+cd MeiliSearch
 ```
 
-Inside the folder, let's compile MeiliSearch.
+Inside the folder, compile MeiliSearch.
 
 ```bash
 # Production version
-cargo run --release
+cargo build --release
+
 # Debug version
-cargo run
+cargo build
 ```
+::: tip
+Compiling in release mode is longer than in debug mode but the binary will be significantly faster. You **must** run a release binary when using MeiliSearch in production.
```

Comment:
```suggestion
Compiling in release mode is longer than in debug mode but the binary will be significantly faster. You **should** run a release binary when using MeiliSearch in production.
```

Reply:
You must use ca laisse sous entendre que le mec n'a techniquement pas le droit. Ce qui est faux. Mais il ne **devrait** pas